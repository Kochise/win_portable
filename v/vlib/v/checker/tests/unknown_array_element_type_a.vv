fn main() {
	a := []abc{}
	println(a)
}
