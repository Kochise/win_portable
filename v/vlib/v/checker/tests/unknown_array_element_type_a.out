vlib/v/checker/tests/unknown_array_element_type_a.vv:2:9: error: unknown type `abc`
    1 | fn main() {
    2 |     a := []abc{}
      |            ~~~
    3 |     println(a)
    4 | }
